readme.md

```diff
@@ -430,6 +430,7 @@
 - [MDBootstrap](https://github.com/mdbootstrap/awesome-mdbootstrap#readme) - Templates, layouts, components, and widgets to rapidly build websites.
 - [Master CSS](https://github.com/master-co/awesome-master-css#readme) - A virtual CSS language with enhanced syntax.
 - [Hydrogen](https://github.com/shopify/awesome-hydrogen#readme) - Edge-first framework for building Shopify storefronts with React.
+- [Tiny JS](https://github.com/thoughtspile/awesome-tiny-js#readme) - Frontend libraries smaller than 2Kb.
```

Comment:
`Kb` is not a valid unit. I think you meant the SI unit `kB`.

Reply:
Same applies to the descriptions in your list.

Reply:
Thanks, shame on me. Should also be preceded by a space. Fixed everywhere.